alpine/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM puppet/puppet-agent-alpine
+
+RUN apk add --update git \
+    && git clone https://github.com/lingua-pupuli/puppet-editor-services.git
+
```

Comment:
Need a `EXPOSE 8082` or some other port here

Reply:
Have we tried STDIO?

Reply:
@jpogran fair point.